pkgs/games/redeclipse/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub, fetchurl, fetchpatch
+, curl, ed, pkgconfig, zlib, libX11
+, SDL2, SDL2_image, SDL2_mixer
+}:
+
+stdenv.mkDerivation rec {
+  pname = "redeclipse";
+  version = "1.6.0";
+
+  src = fetchurl {
+    url = "https://github.com/red-eclipse/base/releases/download/v1.6.0/redeclipse_1.6.0_nix.tar.bz2";
+    sha256 = "0j98zk7nivdsap4y50dlqnql17hdila1ikvps6vicwaqb3l4gaa8";
+  };
+
+  buildInputs = [
+    curl ed pkgconfig libX11 zlib
+    SDL2 SDL2_image SDL2_mixer
+  ];
+
+  makeFlags = [ "-C src/" "prefix=$(out)" ];
```

It does not matter at the moment but we might be more strict in the future.

```suggestion
  makeFlags = [ "-C" "src/" "prefix=$(out)" ];
```